packages/eventual-send/package.json

```diff
@@ -18,6 +18,12 @@
     "lint:types": "tsc",
     "lint:eslint": "eslint '**/*.js'"
   },
+  "exports": {
+    "./package.json": "./package.json",
+    ".": "./src/no-shim.js",
+    "./shim.js": "./shim.js",
+    "./utils.js": "./utils.js"
+  },
```

Comment:
Reviewers, is it ok to add this? It will break any deep imports to other names.

Reply:
Looks like there are no deep imports in agoric-sdk, so I would tentatively say yes.  Maybe ensure that it's in a separate commit, and mark it as a breaking change like `chore(eventual-send)!: restrict exports`.

Reply:
Done